Volume/Models/WeeklyDebrief.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  WeeklyDebrief.swift
+//  Volume
+//
+//  Created by Hanzheng Li on 11/17/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+
+struct WeeklyDebrief: Codable {
+    let creationDate: Date
+    let expirationDate: Date
+    let numShoutouts: Int
+    let numReadArticles: Int
+    let numBookmarkedArticles: Int
+    let readArticleIDs: [ArticleID]
+    let randomArticleIDs: [ArticleID]
```

Comment:
Is it just "random"? Or is it "suggested"? Random is fine I'm just wondering if it's more specific than that.

Reply:
yep, they're literally random at the moment.